### PR TITLE
Error handler

### PR DESCRIPTION
**Pull request by kekuso**
Add some error handler modules to keep code DRY and in line with rpcRunner thrown errors.

### CODE REVIEW

File: prototypes/NilNaNErrorHandler.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const _ = require('lodash');
+
+const NilNaNErrorHandler = (arg) => {
```
Comment (mfuroyama) [inline prototypes/NilNaNErrorHandler.js]: Ugh, sorry...I really hate to be nit-picky, but could you camelCase the function name here? (and below in the exports) `nilNaNErrorHandler`

Comment (kekuso) [inline prototypes/NilNaNErrorHandler.js]: absolutely!

**Comment by kekuso** [comment]
can do! I'll start on that and try finish by tomorrow.

File: prototypes/nullArrayParamErrorHandler.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+// This error handler will only catch a declared null or undefined argument.
+// Note that an empty array parameter WILL pass.
+const nullArrayErrorHandler = (arg) => {
+    // whitelist number, string, and empty array
+    if ((typeof arg[0] === 'number' && !isNaN(arg[0])) || typeof arg[0] === 'string' || (Array.isArray(arg) && arg.length === 0)) {
```
Comment (EdisonTing) [inline prototypes/nullArrayParamErrorHandler.js]: May be use _.isNumber _.isString function in lodash [isNumber](https://lodash.com/docs/4.17.4#isNumber)?

Comment (kekuso) [inline prototypes/nullArrayParamErrorHandler.js]: I'll check it out, thx!

Comment (EdisonTing) [inline prototypes/nullArrayParamErrorHandler.js]: is arg will be always length of 1?

File: prototypes/nullArrayParamErrorHandler.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+        return arg;
+    }
+    throw Error('Argument must be a STRING or NUMBER, not null.');
```
Comment (EdisonTing) [inline prototypes/nullArrayParamErrorHandler.js]: should the message says empty array?

File: prototypes/nullArrayParamErrorHandler.js (hunk 2 above)
Comment (kekuso) [inline prototypes/nullArrayParamErrorHandler.js]: yes, or 0

File: prototypes/nullArrayParamErrorHandler.js (hunk 3 above)
Comment (kekuso) [inline prototypes/nullArrayParamErrorHandler.js]: I'll change it to also accept empty array

**Pull request merged**